release-notes/9.0/preview/api-diff/preview1/Microsoft.NETCore.App/9.0-preview1_System.Threading.md

```diff
@@ -0,0 +1,20 @@
+# System.Threading
+
+``` diff
+ namespace System.Threading {
++    public sealed class Lock {
```

Comment:
The Lock type has a RequiresPreviewFeatures attribute on it:
https://github.com/dotnet/runtime/blob/be5694d8e8494f533bfe5d48d987f5e050df80d2/src/libraries/System.Runtime/ref/System.Runtime.cs#L15088

Is that supposed to be shown here?

Otherwise, System.Threading LGTM

Reply:
Unsure why it was not added, but I created an issue to track this and I added the attribute manually. https://github.com/dotnet/arcade/issues/14498